### PR TITLE
Added spinner/upload message when uploading an image to a post's content

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Adds an upload spinner to post body images being uploaded.

## QA Instructions, Screenshots, Recordings

Jun-29-2020 12-42-46

image

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [x] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Two animated cats spinning on Roombas

### CODE REVIEW

**Comment by nickytonline** [comment]
Just adding some tests before taking this out of draft.

File: app/javascript/crayons/index.js [hunk 1]
```diff
@@ -3,3 +3,4 @@ export * from '@crayons/ButtonGroup';
 export * from '@crayons/Dropdown';
 export * from '@crayons/formElements';
 export * from '@crayons/Modal';
+export * from '@crayons/Spinner';
```
Comment (nickytonline) [inline app/javascript/crayons/index.js:6]: Allows us to import via the [webpack alias](https://webpack.js.org/configuration/resolve/#resolvealias) `@crayons`, e.g. `import { Spinner } from '@crayons';`

**Comment by jacobherrington** [comment]
Does this close https://github.com/thepracticaldev/dev.to/issues/7907?

**Comment by nickytonline** [comment]
> Does this close #7907?

Partially. This was only for the post content image uploads. We'd need to do the same for cover images. I'll do that in another PR and then we can close #7907. Also, I wasn't aware there was an issue open. Was just going off of @ludwiczakpawel's designs. 🙃

**Pull request merged**